specs/external-app/appdetail-resource.yaml

```diff
@@ -117,11 +132,12 @@ paths:
           required: true
           schema:
             type: string
-        - name: namespace
+        - name: appId
           in: query
           required: true
           schema:
             type: string
+            description: pipe separated values of - clusterId, namespace, releaseName(in the same order)
```

Comment:
`pipe separated values of - clusterId, namespace, releaseName` user doesn't known about this logic. 
we pass appId to front-ent, user must reuse `appId`